Research Projects
======

With the aim of quantifying, predicting and controlling cancer evolution, here are the ongoing projects in the lab.

PhD Dr BRCA: Phenotypic Diversity and Dynamics of Resistance in Breast Cancer
------
Using single-cell technologies, we aim to phenotypically describe cells according to what they do, rather than what they are, to then quantify phenotypic diversity. We are validating specific signatures adequate to single-cell data, via exogenous inductions of selected key cell functions in vitro. We aim to understand how plasticity and genetic instability shape the dynamics of resistance, using in vitro models of triple negative breast cancers. This is a collaboration with the Puisieux team (CRCL / Institut Curie).

Relevant publications:
[Assessing Cell Activities rather than Identities to Interpret Intra-Tumor Phenotypic Diversity and Its Dynamics](https://www.cell.com/iscience/fulltext/S2589-0042(20)30246-7). <i>iScience, 2020</i>.
[A stemness-related ZEB1-MSRB3 axis governs cellular pliancy and breast cancer genome stability](https://pubmed.ncbi.nlm.nih.gov/28394329/). <i>Nature Medicine, 2017</i>.

EMMA-STON: Evolutionary Mechanisms of Metabolic Adaptation and Scheduling of Treatment in Oncology
------
We aim to investigate how the evolutionary characteristics of different triple negative breast cancers models (mutations rates, genomic instability, plasticity) dictate the response to therapeutic pressures targetting cell metabolism. We combine in vitro approaches, bioinformatics analyses and mathematical modeling, to optimise the scheduling of synergic drug combinations. This is a collaboration with experimental biology (A Vigneron, CRCL) and mathematics research groups ([E Grenier, INRIA / ENS Lyon](http://www.umpa.ens-lyon.fr/recherche/numed/equipe-projet-inria-numed?set_language=en&cl=en)), whose ultimate goal is to personalise the scheduling of metabolism-targeting therapies, based on the intrinsic evolutionary characteristics of each cancer.

DySECT: Dynamics of Somatic Evolution and Cancerous Trajectories
------
We aim understand the genetic makeup of normal oral squamous mucosa as tissue ages, in order to model its genetic composition over time for both healthy and malignant evolutionary outcomes. This integrates a larger effort led by the [Saintigny Team](http://www.crcl.fr/761-Emerging-Team-Integrated-analysis-of-the-dynamics-of-cancer.crcl.aspx?language=en-GB) and European colleagues to understand the dynamics of progression, from normal to pre-malignancy to cancer in oral cavity cancers, and identify novel treatment and surveillance strategies.

Relevant publications:
[Quantifying local malignant adaptation in tissue‐specific evolutionary trajectories by harnessing cancer’s repeatability at the genetic level](https://onlinelibrary.wiley.com/doi/full/10.1111/eva.12781?utm_campaign=Feed%3A+EvolutionaryApplications+%28Evolutionary+Applications%29). <i>Evolutionary Applications, 2019</i>.
[Separating the Local and Malignant Dimensions of Cancer Adaptation](https://journals.sagepub.com/doi/full/10.1177/1176935119872954). <i>Cancer Informatics, 2019</i>.
